src/frontend/web_application/webpack/config.js

```diff
@@ -7,6 +7,49 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const StyleLintPlugin = require('stylelint-webpack-plugin');
 const clientOptions = require('../config/client.default.js');
 
+const isDev = process.env.NODE_ENV === 'development';
+
+const configureSrcTsLoader = ({ isNode } = { isNode: false }) => {
+  return {
+    module: {
+      rules: [
+        {
+          test: /(?<!\.worker)\.(j|t)sx?$/,
```

Comment:
Hence, no file renaming ?

Reply:
I have a lot of diff that is already conflicts with recent changes (prettier) I prefer keep conventionnal file naming